config/src/config/secure_backend_config.rs

```diff
@@ -45,6 +45,8 @@ pub struct VaultConfig {
     /// renewed up to that many seconds more. If this is not specified, the lease will not
     /// automatically be renewed.
     pub renew_ttl_secs: Option<u32>,
+    /// Network Timeout in milliseconds.
+    pub network_timeout_ms: u64,
```

Comment:
I think making this non-optional is dangerous as it breaks existing configs that are outside the scope of this code base :/. I'll try to commandeer. Thanks for pushing on this!